package.json

```diff
@@ -21,7 +21,7 @@
     "@tailwindcss/forms": "0.5.3",
     "@tailwindcss/line-clamp": "0.4.2",
     "@tailwindcss/typography": "0.5.9",
-    "@teamshares/shoelace": "1.0.1-alpha.4",
+    "@teamshares/shoelace": "1.0.2",
```

Comment:
Would be too late for now if we've already published to npm, but long term what do you think about keeping the core version number in sync w/ that of the upstream shoelace, and incrementing the `xx` in `-alpha.xx` to indicate changes to _just_ the teamshares portion? Then it'd be super easy to see at a glance which version of the upstream shoelace features we should be supporting..

Reply:
We could do that, although if we start adding our own components, and then have breaking changes to them, we'd lose the ability to have control over that. My solution was to add `upstreamVersion` to package.json, and to display that on the docs next to our version number. (That change is awaiting this PR, since I don't want the docs to be ahead of the library).